CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+# Contributor Covenant Code of Conduct
```

Comment:
did you copy this from some other open source project? we're not this stiff, please remove this entirely